### PR TITLE
Fixed some merge issues

### PR DESCRIPTION
**Pull request by zsalch**
Fixed some merge issues.

### CODE REVIEW

**Comment by OlivierBlanvillain** [comment]
@zsalch LGTM

File: src/main/java/featurecat/lizzie/analysis/Leelaz.java [hunk 1]
```diff
@@ -114,10 +115,6 @@ public void startEngine(String engineCommand) throws IOException {
       return;
     }
 
-    File startfolder = new File(config.optString("engine-start-location", "."));
-    String networkFile = config.getString("network-file");
-    // substitute in the weights file
-    engineCommand = engineCommand.replaceAll("%network-file", networkFile);
```
Comment (infinity0) [inline src/main/java/featurecat/lizzie/analysis/Leelaz.java]: Why was this removed?

**Comment by infinity0** [comment]
Can your code review standards be a bit stronger? "Fixed some merge issues" is a bad PR description, and LGTM in response to that is not doing proper code review.

File: src/main/java/featurecat/lizzie/analysis/Leelaz.java (hunk 1 above)
Comment (infinity0) [reply src/main/java/featurecat/lizzie/analysis/Leelaz.java]: Specifically I mean the `%network-file` part. I can see the other parts were moved elsewhere so those seem fine.

Comment (OlivierBlanvillain) [reply src/main/java/featurecat/lizzie/analysis/Leelaz.java]: Seems redundant with what's done in the constructor.

**Comment by OlivierBlanvillain** [comment]
@infinity0 Yeah, we're getting there...

For context this project got to this state without any code review, tests or CI. This PR fixes stuff I introduced in 43153836698eb0b3347a903e66dba533c5ce12d2 which was accidentally merge without being review (except for what you pointed out, which I overlooked). If you want to give us a hand feel to review whatever is in the PR queue!
